sql-statements/sql-statement-kill.md

```diff
@@ -38,6 +36,7 @@ Query OK, 0 rows affected (0.00 sec)
 
 ## MySQL compatibility
 
+* By design, this statement is not compatible with MySQL by default. This helps prevent against a case of a connection being terminated on the wrong TiDB server, since it is common to place multiple TiDB servers behind a load balancer.
```

Comment:
```suggestion
* By design, this statement is not compatible with MySQL by default. This helps prevent against a case of a connection being terminated on the wrong TiDB server, because it is common to place multiple TiDB servers behind a load balancer.
```